tests/test_ring_seeds.py

```
@@ -0,0 +1,99 @@
+                      ring_state={0: 0}, cluster_size=3, rel_energy_kcal=0.0, weight=0.7),
+        RingStateSeed(atoms=mol_to_ase_atoms(molh, cids[1]), energy_eV=-999.9,
+                      ring_state={0: 1}, cluster_size=1, rel_energy_kcal=0.5, weight=0.3),
+    ]
+    return HarvestResult(seeds=seeds, diagnostics={"n_walkers": 2, "n_clusters": 2,
+                                                   "stereo_reject_rate": 0.0})
+
+
+def test_seed_roundtrip(tmp_path):
+    mol = Chem.MolFromSmiles("CC1CCCCC1")
+    molh = Chem.AddHs(mol)
+    p = AllChem.ETKDGv3(); p.randomSeed = 1
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)